global-variables.asciidoc

```diff
@@ -15,7 +15,9 @@ pipeline {
     agent any
     
     environment {
-        FOO="foo"
+        FOO = "foo"
+        AWS_ACCESS_KEY_ID     = credentials('jenkins_aws_secret_key_id')
```

Comment:
TIL! 
didn't know this syntax... 

what credential type does it need? secret text?

Reply:
> Jenkins' declarative Pipeline syntax has the credentials() helper method (used within the environment directive) which supports secret text, username and password, as well as secret file credentials

Source: https://jenkins.io/doc/book/pipeline/jenkinsfile/

Reply:
🎉